examples/attributes/attributes-element.js

```diff
@@ -11,10 +11,12 @@ class AttributesElement extends TemplateElement {
 
     template() {
         const templateResult = html`
-            <div foo="bar" class="before:block before:content-[&apos;Festivus&apos;]">no interpolation</div>
-            <div foo="${'bar'}">single attribute interpolation</div>
-            <div foo="${'bar'}" bar="${'baz'}">multiple attribute interpolations</div>
-            <div foo="${'bar'} baz ${'foo'}">multiple interpolations in single attribute</div>
+            <div foo1="bar" class="before:block before:content-['Festivus']">no interpolation</div>
+            <div foo2="${'bar'}">single attribute interpolation</div>
+            <div foo3="${'bar'}" bar="${'baz'}">multiple attribute interpolations</div>
+            <div foo4="${'bar'} baz ${'foo'} before:content-['Festivus']">
```

Comment:
what about the unquoted case ?
https://arc.net/l/quote/neouemuc

do we still test this somewhere ?

Reply:
And what exatly is this. 
baz ${'foo'}
Just having a single/static interpolation somewhere through ${'foo'} ?

I mean .. it's cool but isn't it the same as conditional attributes ?

Reply:
This is just an example element for debugging things in the browser through the dev server. The unquoted case is tested somewhere else.
baz ${'foo'} is showing what it looks like when attributes have interpolations inside static parts of the attribute value.